ethcore/res/ethereum/tests-issues/currents.json

```diff
@@ -0,0 +1,215 @@
+                                "CreateOOGafterInitCodeRevert_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata3_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeRevert2_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCode_d0g1v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndataSize_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCode_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata2_d0g0v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata_d0g1v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata2_d0g1v0_Constantinople",
+                                "CreateOOGafterInitCodeReturndata_d0g0v0_Constantinople"
+                        ]
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stEIP158Specific",
+                        "subtests": [
+                                "callToEmptyThenCallError_d0g0v0_Byzantium",
+                                "callToEmptyThenCallError_d0g0v0_Constantinople",
+                                "callToEmptyThenCallError_d0g0v0_EIP158"
+                        ]
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stPreCompiledContracts",
+                        "subtests": [
+                                "identity_to_smaller_d0g0v0_Constantinople",
+                                "identity_to_bigger_d0g0v0_Constantinople"
+                        ]
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stReturnDataTest",
+                        "subtests": [
+                                "modexp_modsize0_returndatasize_d0g1v0_Constantinople",
+                                "modexp_modsize0_returndatasize_d0g2v0_Constantinople",
+                                "modexp_modsize0_returndatasize_d0g3v0_Constantinople"
+                        ]
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stRevertTest",
+                        "subtests": [
+                                "RevertOnEmptyStack_d0g0v0_Constantinople"
+                        ]
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stSpecialTest",
+                        "subtests": [
+                                "FailedCreateRevertsDeletion_d0g0v0_Constantinople",
+                                "push32withoutByte_d0g0v0_Constantinople"
+                        ]
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stSystemOperationsTest",
```

Comment:
This can be removed as well with https://github.com/paritytech/parity-ethereum/pull/9577/commits/544f6a3d0c9fa61f626b8bb3d137f7c5039d7af7